README.md

```diff
@@ -1 +1,11 @@
 # hw01
+
+## Self-introduction
+1. My name is **Rui Tang**. 
+2. I am a *MAPSS* student in _Psychology_.
```

Comment:
Nice job with consistent italics to refer to programs after using bold for your name